make dev fails when GOPATH includes multiple paths

The make dev fails on https://github.com/hashicorp/nomad-pack/blob/main/GNUmakefile#L76 if GOPATH includes multiple paths like "/path/1:/path/2". 

Such GOPATH is valid as described here: https://go.googlesource.com/vscode-go/+/refs/heads/dev.go2go/docs/gopath.md


The error I see:
```
==> Building nomad-pack...
cp: /Users/gucio/dev:/usr/local/go/bin/nomad-pack: No such file or directory
make: *** [dev] Error 1
```

caused by `GOPATH=${HOME}/dev:/usr/local/go`. Also note that `go install` behaves correctly.

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -69,10 +69,16 @@ hclfmt: ## Format HCL files with hclfmt
 
 .PHONY: dev
 dev:
+ifneq (,$(shell echo ${PACK_INSTALL_DEV}))
+	@echo "==> Building and installing nomad-pack..."
+	@CGO_ENABLED=0 go build -ldflags $(GO_LDFLAGS) -o ./bin/nomad-pack
+	@CGO_ENABLED=0 go install -ldflags $(GO_LDFLAGS)
+	@echo "==> Done"
+else
 	@echo "==> Building nomad-pack..."
 	@CGO_ENABLED=0 go build -ldflags $(GO_LDFLAGS) -o ./bin/nomad-pack
-	@CGO_ENABLED=0 go install -ldflags $(GO_LDFLAGS) .
 	@echo "==> Done"
+endif
 
 pkg/%/nomad-pack: GO_OUT ?= $@
 pkg/windows_%/nomad-pack: GO_OUT = $@.exe

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -68,12 +68,10 @@ hclfmt: ## Format HCL files with hclfmt
 	@if (git status -s | grep -q -e '\.hcl$$' -e '\.nomad$$' -e '\.tf$$'); then echo the following HCL files are out of sync; git status -s | grep -e '\.hcl$$' -e '\.nomad$$' -e '\.tf$$'; exit 1; fi
 
 .PHONY: dev
-dev: GOPATH=$(shell go env GOPATH)
 dev:
 	@echo "==> Building nomad-pack..."
 	@CGO_ENABLED=0 go build -ldflags $(GO_LDFLAGS) -o ./bin/nomad-pack
-	@rm -f $(GOPATH)/bin/nomad-pack
-	@cp ./bin/nomad-pack $(GOPATH)/bin/nomad-pack
+	@CGO_ENABLED=0 go install -ldflags $(GO_LDFLAGS) .
 	@echo "==> Done"
 
 pkg/%/nomad-pack: GO_OUT ?= $@

